src/FluentNHibernate/FluentNHibernate.csproj

```diff
@@ -13,22 +13,26 @@
     <Description>FluentNHibernate</Description>
   </PropertyGroup>
   
-  <Import Project="$(MSBuildThisFileDirectory)..\Shared.msbuild" />
   
   <PropertyGroup>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="NHibernate" Version="5.0.3" />    
-  </ItemGroup>
-  
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+
+  <Import Project="$(MSBuildThisFileDirectory)..\Shared.msbuild" />
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
+    <PackageReference Include="NHibernate" Version="5.0.3" />
     <Reference Include="System.configuration" />
     <Reference Include="System.ServiceModel" />
     <Reference Include="System.Transactions" />
   </ItemGroup>
-  
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="NHibernate" Version="5.1.0" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.4.3" />

```

Comment:
We need System.Data.SqlClient?

Reply:
Yes, it seems it is needed for the [MsSqlConnectionStringBuilder](https://github.com/FluentNHibernate/fluent-nhibernate/blob/master/src/FluentNHibernate/Cfg/Db/MsSqlConnectionStringBuilder.cs#L55)

Reply:
We could create our own implementation of the builder like the MySqlConnectionStringBuilder. Because we use bypass the connectionString, and have no interaction with the sqlconnectionstringbuilder for the user.

Reply:
The use of a package that the developer does not have a interaction with it is not a good implementation. (the user only pass the connectionString as string)

Reply:
@danilobreda Seem right. Do you take care of this in another PR when we finish this one?

Reply:
@jrgcubano yes i can, i will wait for this release.